plugins/utilities/file_share.py

```diff
@@ -33,7 +33,7 @@
 
 app = _babase.app
 
-MODS_DIR = app.python_directory_user
+MODS_DIR = app.env.python_directory_user
```

Comment:
can we keep it so it also works on older 1.7 versions? otherwise it might not be nice to have some plugins only work with the latest 1.7.x versions and other plugins working only for the earlier 1.7.x versions.

looks like old api was [deprecated in build 21282](https://github.com/efroemling/ballistica/blob/master/CHANGELOG.md#1727-build-21282-api-8-2023-08-30), so could be nice to call the newer api method only when client is on a build number after the deprecation happened (can be checked using `ba.env.TARGET_BALLISTICA_BUILD` me thinkypies!).

Reply:
Done.